ui/apps/dashboard/src/queries/useDefaultEventKey.ts

```diff
@@ -0,0 +1,51 @@
+import { graphql } from '@/gql';
+import { useGraphQLQuery } from '@/utils/useGraphQLQuery';
+
+const query = graphql(`
+  query GetEventKeysForBlankSlate($environmentID: ID!) {
+    environment: workspace(id: $environmentID) {
+      ingestKeys(filter: { source: "key" }) {
+        name
+        presharedKey
+        createdAt
+      }
+    }
+  }
+`);
+
+export function useDefaultEventKey({ envID }: { envID: string }) {
+  const res = useGraphQLQuery({
+    query,
+    variables: { environmentID: envID },
+  });
+
+  if (res.data) {
```

Comment:
Maybe we should at least log an error here so that we can distinguish error condition versus no data when troubleshooting.